core/app/models/workarea/search/storefront/product.rb

```diff
@@ -1,7 +1,7 @@
 module Workarea
   module Search
     class Storefront
-      class Product < Search::Storefront
+      class Product < ::Workarea::Search::Storefront
```

Comment:
We're using this same abbreviated syntax in all of the `search/admin` models, I wonder why you didn't need to do this same change in those classes?

Reply:
Maybe something to do with the name `Product`?